### ISSUE TITLE
Test failures

### ISSUE THREAD

**Issue by FRidh** [pyca/pynacl#550]
```
============================= test session starts ==============================
platform linux -- Python 3.7.4, pytest-5.1.0, py-1.8.0, pluggy-0.12.0
hypothesis profile 'default' -> database=DirectoryBasedExampleDatabase('/build/PyNaCl-1.3.0/.hypothesis/examples')
rootdir: /build/PyNaCl-1.3.0
plugins: hypothesis-4.7.3
collected 4618 items

tests/test_aead.py ..................................................... [  1%]
.....................                                                    [  1%]
tests/test_bindings.py ......................                            [  2%]
tests/test_box.py .............F                                         [  2%]
tests/test_encoding.py ......                                            [  2%]
tests/test_exc.py ....                                                   [  2%]
tests/test_generichash.py .............................................. [  3%]
........................................................................ [  5%]
........................................................................ [  6%]
........................................................................ [  8%]
........................................................................ [  9%]
........................................................................ [ 11%]
........................................................................ [ 12%]
........................................................................ [ 14%]
........................................................................ [ 16%]
........................................................................ [ 17%]
........................................................................ [ 19%]
........................................................................ [ 20%]
........................................................................ [ 22%]
........................................................................ [ 23%]
........................................................................ [ 25%]
........................................................................ [ 26%]
........................................................................ [ 28%]
........................................................................ [ 30%]
........................................................................ [ 31%]
........................................................................ [ 33%]
........................................................................ [ 34%]
........................................................................ [ 36%]
..............................................................ss........ [ 37%]
........................................................................ [ 39%]
........................................................................ [ 41%]
........................................................................ [ 42%]
..............................ss........................................ [ 44%]
........................................................................ [ 45%]
........................................................................ [ 47%]
........................................................................ [ 48%]
....                                                                     [ 48%]
tests/test_hash.py ........                                              [ 49%]
tests/test_hashlib_scrypt.py ....                                        [ 49%]
tests/test_kx.py ......                                                  [ 49%]
tests/test_public.py ...............                                     [ 49%]
tests/test_pwhash.py ................................................... [ 50%]
..................................................                       [ 51%]
tests/test_sealed_box.py ........F...                                    [ 52%]
tests/test_secret.py ........F.                                          [ 52%]
tests/test_secretstream.py .....                                         [ 52%]
tests/test_shorthash.py ................................................ [ 53%]
........................................................................ [ 54%]
.........                                                                [ 55%]
tests/test_signing.py .................................................. [ 56%]
........................................................................ [ 57%]
........................................................................ [ 59%]
........................................................................ [ 60%]
........................................................................ [ 62%]
........................................................................ [ 64%]
........................................................................ [ 65%]
........................................................................ [ 67%]
........................................................................ [ 68%]
........................................................................ [ 70%]
........................................................................ [ 71%]
........................................................................ [ 73%]
........................................................................ [ 74%]
........................................................................ [ 76%]
........................................................................ [ 78%]
........................................................................ [ 79%]
........................................................................ [ 81%]
........................................................................ [ 82%]
........................................................................ [ 84%]
........................................................................ [ 85%]
........................................................................ [ 87%]
........................................................................ [ 88%]
........................................................................ [ 90%]
........................................................................ [ 92%]
........................................................................ [ 93%]
........................................................................ [ 95%]
........................................................................ [ 96%]
........................................................................ [ 98%]
........................................................................ [ 99%]
F                                                                        [ 99%]
tests/test_utils.py ...                                                  [100%]

=================================== FAILURES ===================================
_______________________________ test_wrong_types _______________________________

    def test_wrong_types():
        priv = PrivateKey.generate()

        check_type_error(("PrivateKey must be created from a 32 bytes long "
                          "raw secret key"),
>                        PrivateKey, 12)

tests/test_box.py:279:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

expected = 'PrivateKey must be created from a 32 bytes long raw secret key'
f = <class 'nacl.public.PrivateKey'>, args = (12,)
e = <ExceptionInfo TypeError tblen=2>
@py_assert4 = '<ExceptionInfo TypeError tblen=2>', @py_assert1 = False

    def check_type_error(expected, f, *args):
        with pytest.raises(TypeError) as e:
            f(*args)
>       assert expected in str(e)
E       AssertionError: assert 'PrivateKey must be created from a 32 bytes long raw secret key' in '<ExceptionInfo TypeError tblen=2>'
E        +  where '<ExceptionInfo TypeError tblen=2>' = str(<ExceptionInfo TypeError tblen=2>)

tests/test_box.py:271: AssertionError
_______________________________ test_wrong_types _______________________________

    def test_wrong_types():
        priv = PrivateKey.generate()

        check_type_error(("SealedBox must be created from a PublicKey"
                          " or a PrivateKey"),
>                        SealedBox, priv.encode())

tests/test_sealed_box.py:120:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

expected = 'SealedBox must be created from a PublicKey or a PrivateKey'
f = <class 'nacl.public.SealedBox'>
args = (b'\x0e1\x0c\x13\xc1,R\xe1\xf4:96\xb6\x82;\x15DU\xbc\xffsKP\xc7\xecCR\x07I\x16\xa4\xbb',)
e = <ExceptionInfo TypeError tblen=2>
@py_assert4 = '<ExceptionInfo TypeError tblen=2>', @py_assert1 = False

    def check_type_error(expected, f, *args):
        with pytest.raises(TypeError) as e:
            f(*args)
>       assert expected in str(e)
E       AssertionError: assert 'SealedBox must be created from a PublicKey or a PrivateKey' in '<ExceptionInfo TypeError tblen=2>'
E        +  where '<ExceptionInfo TypeError tblen=2>' = str(<ExceptionInfo TypeError tblen=2>)

tests/test_sealed_box.py:112: AssertionError
_______________________________ test_wrong_types _______________________________

    def test_wrong_types():
        box = SecretBox(b"11" * 32, encoder=HexEncoder)

        check_type_error("SecretBox must be created from 32 bytes",
>                        SecretBox, 12)

tests/test_secret.py:152:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

expected = 'SecretBox must be created from 32 bytes'
f = <class 'nacl.secret.SecretBox'>, args = (12,)
e = <ExceptionInfo TypeError tblen=2>
@py_assert4 = '<ExceptionInfo TypeError tblen=2>', @py_assert1 = False

    def check_type_error(expected, f, *args):
        with pytest.raises(TypeError) as e:
            f(*args)
>       assert expected in str(e)
E       AssertionError: assert 'SecretBox must be created from 32 bytes' in '<ExceptionInfo TypeError tblen=2>'
E        +  where '<ExceptionInfo TypeError tblen=2>' = str(<ExceptionInfo TypeError tblen=2>)

tests/test_secret.py:145: AssertionError
_______________________________ test_wrong_types _______________________________

    def test_wrong_types():
        sk = SigningKey.generate()

        check_type_error("SigningKey must be created from a 32 byte seed",
>                        SigningKey, 12)

tests/test_signing.py:196:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

expected = 'SigningKey must be created from a 32 byte seed'
f = <class 'nacl.signing.SigningKey'>, args = (12,)
e = <ExceptionInfo TypeError tblen=2>
@py_assert4 = '<ExceptionInfo TypeError tblen=2>', @py_assert1 = False

    def check_type_error(expected, f, *args):
        with pytest.raises(TypeError) as e:
            f(*args)
>       assert expected in str(e)
E       AssertionError: assert 'SigningKey must be created from a 32 byte seed' in '<ExceptionInfo TypeError tblen=2>'
E        +  where '<ExceptionInfo TypeError tblen=2>' = str(<ExceptionInfo TypeError tblen=2>)

tests/test_signing.py:189: AssertionError
=============================== warnings summary ===============================
/nix/store/7yy89ic51jmxdld7v2cbyg6vzylpmfqx-python3.7-hypothesis-4.7.3/lib/python3.7/site-packages/hypothesis/_settings.py:193
  /nix/store/7yy89ic51jmxdld7v2cbyg6vzylpmfqx-python3.7-hypothesis-4.7.3/lib/python3.7/site-packages/hypothesis/_settings.py:193: HypothesisDeprecationWarning: The timeout setting can safely be removed with no effect.
    verbosity=self.verbosity,

-- Docs: https://docs.pytest.org/en/latest/warnings.html
============ 4 failed, 4610 passed, 4 skipped, 1 warnings in 24.00s ============
```

**Comment by alex** [comment]
This should be fixed by #546

**Comment by FRidh** [comment]
okay, thank you. Is there any with those fixes planned?
